frontend/src/hooks/useCreateBatch.ts

```diff
@@ -1,3 +1,5 @@
+// NOTE: THIS HOOK IS NOT CURRENTLY USED/CALLED ANYWHERE
+// SEE useProjectStore FOR EQUIVALENT FUNCTIONALITY THAT IS ACTUALLY CALLED
```

Comment:
Is this going to be used in the original pr or something?

Reply:
In the original PR i kind of left it there as a reference for how I had tried to do this stuff as a hook (but I couldn't call it properly). I had left it in there in case reviews indicated it should be used if how I had done it in the store wasn't suitable. Kind of like 'this was an alternative solution attempt that didn't pan out but could be brought back if necessary'. But seeing as there hasn't been any comments indicating to use the hook instead of what I did in the store I will just delete it.